LetsDoIt.CustomValueTypes/src/LetsDoIt.CustomValueTypes/Email.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Net.Mail;
+using System.Text.Json.Serialization;
+
+namespace LetsDoIt.CustomValueTypes
+{
+
+    [JsonConverter(typeof(EmailJsonConverter))]
+    public readonly struct Email
```

Comment:
Why do we need to make the struct readonly?

Reply:
So that members can be accessed only through the constructor where our validation occurs.